src/Service/Lambda/src/Input/PublishLayerVersionRequest.php

```diff
@@ -196,13 +197,19 @@ public function setLicenseInfo(?string $value): self
 
     public function validate(): void
     {
-        foreach (['LayerName', 'Content'] as $name) {
-            if (null === $this->$name) {
-                throw new InvalidArgument(sprintf('Missing parameter "%s" when validating the "%s". The value cannot be null.', $name, __CLASS__));
-            }
+        if (null === $this->LayerName) {
+            throw new InvalidArgument(sprintf('Missing parameter "LayerName" when validating the "%s". The value cannot be null.', __CLASS__));
+        }
+
+        if (null === $this->Content) {
+            throw new InvalidArgument(sprintf('Missing parameter "Content" when validating the "%s". The value cannot be null.', __CLASS__));
         }
-        if ($this->Content) {
```

Comment:
I also refactor how requires properties are handled in validation: I replace `foreach (['foo', 'bar'` by `if (foo... if (bar...`.

That allows me to group similar validation on same properties and fix useless check like this one.

Reply:
Cool. I see this now.